README.md

```diff
@@ -139,75 +139,137 @@ tool for the simulation and drive the vehicle towards a wall.
+```bash
+# On the localhost
+gordon@f1sim:~$ cd sim_ws/src/
+gordon@f1sim:~sim_ws/src/$ cp -r ~/git/f1tenth_labs_openrepo/f1tenth_lab2/safety_node .
+gordon@f1sim:~sim_ws/src/$ ls
+f1tenth_gym_ros safety_node
+```
+### 5.1 Personalizing your Node by Name 
+You cannot have more than one node of the same name built and sourced to run at one time in ROS.
+You will need to change the name of the node to fit the following convention: 
+**student's_name**_*safety_node*. 
+
+You will need to edit or change the following within the copied package:
+1. package.xml
+2. CMakeLists.txt 
+3. setup.py
+4. setup.cfg
+5. safety_node/ to **student's_name**_*safety_node*
+6. Both the cpp file and python file to **student's_name**_*safety_node*.py/cpp
+
+An example package will be uploaded to this repository that will show changes required 
+for an example node **bree_safety_node**.
 
 Develop your solution directly in the simulation container, with your
 package in `/sim_ws/src` alongside the simulation package. To do so,
 edit `docker-compose.yml` in the `f1tenth_gym_ros` container.
 
 ```yaml
 # sim_ws/src/f1tenth_gym_ros/docker-compose.yml
+# Do for each node you create in this course
 <<< SNIP >>>
     build: ./
     volumes:
       - .:/sim_ws/src/f1tenth_gym_ros                             # this should have been modified already
-      - <abspath>/sim_ws/src/safety_node:/sim_ws/src/safety_node  # Add this line for each node
+      - <abspath>/sim_ws/src/YOUR_NAME_safety_node:/sim_ws/src/YOUR_NAME_safety_node  #Newly added
     environment:
 <<< SNIP >>>
 ```
 
+If the following error is encountered, the safety_node has not been properly included in the container.
+Retry adding it as a mount point and restarting the composition.
+```bash
+#In Container
+root@445b65e71801:/sim_ws# colcon build --packages-select safety_node
+[0.362s] WARNING:colcon.colcon_core.package_selection:ignoring unknown package ’safety_node’ in --packages-select
+```
 Note that if you're using Windows, make sure your files have Unix
 style line endings. You can use `dos2unix` or have correct settings in
 your text editor.
 
+## 6. Physical Vehicle Changes
+Given the maximum speed of 60 miles per hour on the vehicle, the 
+maximum permissible TTC value is 1.5 seconds.
+
+Within the simulator, the odometry topic is published to **/ego_racecar/odom**, 
+on the vehicle the topic is published to **/odom**. Before deploying to 
+the vehicle, subscriptions must be updated.
+
+You must be able to change the parameter on the command line between the two for 
+your implementation. You should be able to run something similar to the following 
+command in your launch file:
+
+```bash
+ros2 launch safety_node_launch.py bree_safety_node:=true TTC:=2.0 odom_pub:="/odom"
+```
+
+
+
+
+*~!!!!! DR.TESSLER- I want to double check this launch command before we post this. I 
+do not have access to the correct command on my laptop, but I do know the first 2 parameters 
+were correct~!!!!!!*
```

Comment:
I would make this a little simpler:
`ros2 launch safety_node_launch.py mode:=[sim|vehicle] student:=STUDENTNAME`
eg
`ros2 launch safety_node_launch.py mode:="sim" student:="bree" ttc:=2.0

Reply:
Okay! This has been implemented :)